_ml-commons-plugin/conversational-search.md

```diff
@@ -275,6 +278,8 @@ Use the following steps to set up an HTTP connector using the OpenAI GPT 3.5 mod
     "description": "This is a public model group"
   }
   ```
+  {% include copy-curl.html %}
```

Comment:
On line 272, at and after the step "2. Create a new model group for the connected model. You'll use the `model_group_id` returned by the Register API to register the model:" the numbering is incorrect (because of tabs)